tests/testthat/test-workflow_set.R

```diff
@@ -169,11 +175,29 @@ test_that("workflow_set can handle case weights", {
          workflow_map(
             "fit_resamples",
             resamples = vfold_cv(cars, v = 5)
-         )},
-      error = TRUE
-   )
+         )
+   })
+
+   class_note <- extract_workflow_set_result(car_set_2, "reg_lm") %>%
+      tune::collect_notes() %>%
+      dplyr::select(note)
+
+   expect_match(class_note$note[1], "classed case weights column")
```

Comment:
This hardcoding of the error message is actually what I was trying to avoid. I think you can use `expect_snapshot()` on `class_note$note[1]` to capture the error message in a way that won't fail on CRAN if I ever change this in workflows.

Reply:
Sorry if that wasn't clear. But yea these error messages from workflows look good enough to me, so removing all the error handling from workflowsets is the right move IMO